devbook.xsl

```diff
@@ -568,6 +606,16 @@
       </footer>
       <script src="https://assets.gentoo.org/tyrian/jquery.min.js"></script>
       <script src="https://assets.gentoo.org/tyrian/bootstrap.min.js"></script>
+      <script src="https://unpkg.com/lunr/lunr.js"></script>
```

Comment:
```suggestion
      <script src="https://assets.gentoo.org/lunr/lunr.min.js"></script>
```

Reply:
It is assumed that this file will be uploaded to that location, correct?

Reply:
Yup, I'm currently working on it.

Reply:
The deployment of lunr.min.js to infra is almost ready: https://github.com/gktrk/devmanual-assets